docs/source/ruletypes.rst

```diff
@@ -510,6 +522,74 @@ This value is added in back of the event. For example,
 
 ``kibana4_end_timedelta: minutes: 2``
 
+use_kibana_discover
+^^^^^^^^^^^^^^^^^^^
+
+``use_kibana_discover``: Enables the generation of the ``kibana_link`` variable for a particular version of the Kibana Discover application.
+This setting requires that ``kibana_discover_url`` and ``kibana_discover_index_pattern_id`` are also configured.
+
+The currently supported versions of Kibana Discover are: 
```

Comment:
I decided to go with an explicit version for a couple reasons.  First, the Discover state, while it's been relatively consistent, isn't a well defined public contract.  There will be changes.  They could happen in a minor release with the addition of new functionality.

The second reason is that there are tones of issues asking if the Kibana4 dashboard apps supports the newer versions.  This hopefully clarifies and addresses what is supported.